src/main.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const http = require('http');
+
+const server = http.createServer((req, res) => {
+  res.setHeader('Content-type', 'application/json');
+
+  const paramsSplitted = req.url.slice(1).split('?');
+  const paths = paramsSplitted[0].split('/');
+  const params = new URLSearchParams(paramsSplitted[1]);
+  const errors = [];
+  const query = {};
+
+  for (const key of params) {
+    query[key[0]] = key[1];
+  }
```

Comment:
There is a better way to receive query params:
![image](https://user-images.githubusercontent.com/45945732/234019505-5a2e59ab-ee4f-4b5d-9c00-aea4ad2dbe74.png)